lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -9,10 +21,18 @@ def self.included(base)
         base.send(:private_attribute, :original)
       end
 
+      def translated_attr
+        self.class.read_translated_changeset_attr
```

Comment:
this can just be `self.class.instance_variable_get(:@translated_changeset_attrs)`